cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -190,57 +197,57 @@ data ClientCommand
     (Maybe ExplorerAPIEnpoint)
     [SigningKeyFile]
    deriving Show
-runCommand :: ClientCommand -> ExceptT CliError IO ()
```

Comment:
Is there a better way to introduce `AssociateWithIOCP`? Only one command uses this parameter and I've spent a lot of time removing parameters like this.

Reply:
/cc @karknu , @coot

Reply:
See: https://github.com/input-output-hk/cardano-node/pull/588/files#r383898945